tests/test_dynamicDetection.py

```diff
@@ -57,7 +57,7 @@ def setUp(self):
 
         # Relative tolerance for tweak factor
         # Not sure why this isn't smaller; maybe due to use of Poisson instead of Gaussian noise?
-        self.rtol = 0.1
+        self.rtol = 0.15
```

Comment:
Might be worth a comment in the commit message about this change (not that we can really explain it at the moment...😞).

Reply:
I thought about adding one, but the initial in-line that's already there seems to capture everything I would want to say anyway. In the spirit of adding something, I've added this:
```
# It seems as if some sky objects are being placed in the extra
# background region, which is causing the offset between the expected
# factor and the measured factor to be larger than otherwise expected.
# This relative tolerance was increased from 0.1 to 0.15 on DM-23781 to
# account for this.
```

Reply:
Looks good (I did just mean adding something to the commit message, but this works too!)

Reply:
Ah, the _commit_ message! For some reason I didn't parse that the first time around. Commit message also updated with some background too.